about.md

```diff
@@ -24,30 +24,45 @@ the early planning stages, it will hopefully help keep our dogs well fed, but al
 
 ### [Moo Machine](https://sites.google.com/a/stevenmiller.info/www/home/projects/moomachinehome)
 
-An Altair 8800 style machine designed to implement the ["COW" programming language](http://web.archive.org/web/20130403133623/bigzaphod.org/cow/).  It's a really simplistic language, but is technically Turing complete.  The machine itself is powered by a Arduino Mega (more for the IO pins then the actual processor)  Unfortunately the 
-original site is down, but thanks to the Internet Archive project, we can still view the original page.
+An Altair 8800 style machine designed to implement the ["COW" programming language](https://bigzaphod.github.io/COW/).  It's a really simplistic language, but is technically Turing complete.  The machine itself is powered by a Arduino Mega (more for the IO pins then the actual processor)  
 
 
 
 
 # Professional Experience
 
-### Gordon Food Service (March 2008 - Present)
+### Gordon Food Service - Security Engineer (May 2016 - Present)
 
-Responsible for the management of 170+ remote Linux servers, 20+ Linux/HPUX/AIX/Windows central servers, 700+ Windows 
+Responsible for monitoring and adjusting security controls within Gordon Food
+Service.  Performed risk assessments against various packages software solution.
```

Comment:
does it make sense to have this in past tense when you're talking about your current position?